### PR TITLE
Added new tests for testing error paths WFFHCOHORT-6 (API supports mu…

### PR DESCRIPTION
**Pull request by jaigoradia**
…ltiple measures)  plus updated automation framework to generate formatted results xml file

### CODE REVIEW

File: tests/scenarios/breastCancerTestScenarios.yaml [hunk 1]
```diff
@@ -17,4 +17,4 @@ scenarios:
 
 reporting:
 - module: junit-xml
-  filename: ../results/sampleTest.xml
\ No newline at end of file
+  filename: /bzt-configs/tests/results/sampleTest.xml
```
Comment (deekim) [inline tests/scenarios/breastCancerTestScenarios.yaml:20]: non-blocking request

I'm going to assume this works based on the example in the [README](https://github.com/Alvearie/quality-measure-and-cohort-service/blame/7961c2f5c76e45f10d58bd83f3c8e84369455095/tests/README.md#L37),
but I feel like it should at least be better documented?

**Review by deekim — APPROVED** [review]
lgtm, but I'd be looking to @ToddHicks since I'm not super familiar with these types of configuration files.

File: tests/scenarios/breastCancerTestScenarios.yaml (hunk 1 above)
Comment (ToddHicks) [reply tests/scenarios/breastCancerTestScenarios.yaml:20]: Good point, this change was made so the result xml file would be put in the shared drive in the docker container. Jai, want to add some details in the README about some of these settings?

**Review by ToddHicks — APPROVED** [review]
Changes look good to me.

**Pull request merged**